### ISSUE TITLE
feat: add ability to colorize connections in tabs and on connections screen

### ISSUE THREAD

**Issue by josegonzalez** [Paxa/postbird#13]
This will allow users to understand that the current connection is to:

- production
- readonly
- staging

On the connection screen itself, it can be something like the following (inspired by sequel pro):

![https://cl.ly/1H3Q1q340e45](https://cl.ly/1H3Q1q340e45/Image%202017-05-26%20at%2010.44.09%20PM.png)

For the tabs, either:

- the active tab should have the color specified, and inactive tabs should have it as well, though faded out to show that it's not the active tab.
- there should be a global header that is updated with the current db connection and some presence of that color (not sure what would be aesthetically pleasing).

**Comment by Paxa** [comment]
I think it will be nice, thought I rarely use sequel pro to connect to production

**Comment by josegonzalez** [comment]
Yeah, while I'd hate to have prod connections, if it's necessary (sometimes it is) then having colorization provides a nice way for devs to distinguish whats going on.

I also have stuff like `VPC STAGING READONLY`, so I know I can't do writes to those :)
